src/App.tsx

```diff
@@ -207,17 +207,14 @@ function logLevel(text: string): [string, string, number] {
   const restOfText = words.slice(2).join(' ');
   let debug = "";
   let message = "";
-  let level_val:number = 0;
   if (firstWord in debug_levels) {
-    debug = firstWord;
+    debug = Object.keys(debug_levels).find(key => debug_levels[key] === level_val) || "UNKNOWN";
```

Comment:
Is it easier to reverse the association in your dictionary so it maps from numbers to strings and then just do a lookup here?